appveyor.yml

```diff
@@ -0,0 +1,186 @@
+      EXTRA_TEST_ARGS: "--exclude fortran_compiler"
+
+    - PYTHON: "C:\\Python35-x64"
+      PYTHON_VERSION: "3.5.x"
+      PYTHON_ARCH: "64"
+      CMAKE_GENERATOR: Visual Studio 10
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run-with-visual-studio.cmd"
+      EXTRA_TEST_ARGS: "--exclude fortran_compiler"
+
+    # MinGW (Python 2 & 3, 64 bit only)
+    - PYTHON: "C:\\Python27-x64"
+      PYTHON_VERSION: "2.7.x"
+      PYTHON_ARCH: "64"
+      CMAKE_GENERATOR: "MinGW Makefiles"
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run-with-mingw.cmd"
+      EXTRA_TEST_ARGS: "--exclude fortran_compiler"
+
+    - PYTHON: "C:\\Python35"
+      PYTHON_VERSION: "3.5.x"
+      PYTHON_ARCH: "32"
+      CMAKE_GENERATOR: "MinGW Makefiles"
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run-with-mingw.cmd"
+      EXTRA_TEST_ARGS: "--exclude fortran_compiler"
+
+    # Default -- make sure Visual Studio works as the default
+    # (Python 2 & 3, 64 bit only)
+    - PYTHON: "C:\\Python27-x64"
+      PYTHON_VERSION: "2.7.x"
+      PYTHON_ARCH: "64"
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run-with-visual-studio.cmd"
+      EXTRA_TEST_ARGS: "--exclude fortran_compiler"
+
+    - PYTHON: "C:\\Python35-x64"
+      PYTHON_VERSION: "3.5.x"
+      PYTHON_ARCH: "64"
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run-with-visual-studio.cmd"
+      EXTRA_TEST_ARGS: "--exclude fortran_compiler"
+
+init:
+  - cmd: "ECHO RDP CONNECTION INFORMATION (IN CASE OF HANGING BUILD)"
+  - ps: "\
+      iex (\
+        (New-Object Net.Webclient).DownloadString(\
+          'https://raw.githubusercontent.com/\
+           appveyor/ci/master/scripts/enable-rdp.ps1'\
+        )\
+      )"
+
+install:
+  # If there is a newer build queued for the same PR, cancel this one.
+  # The AppVeyor 'rollout builds' option is supposed to serve the same
+  # purpose but it is problematic because it tends to cancel builds pushed
+  # directly to master instead of just PR builds (or the converse).
+  # credits: JuliaLang developers.
+  - ps: "\
+    if (\
+      $env:APPVEYOR_PULL_REQUEST_NUMBER -and \
+      $env:APPVEYOR_BUILD_NUMBER -ne (\
+        (\
+          Invoke-RestMethod \
+            ` https://ci.appveyor.com/api/projects/\
+                $env:APPVEYOR_ACCOUNT_NAME/\
+                $env:APPVEYOR_PROJECT_SLUG/\
+                history?recordsNumber=50\
+        ).builds | ` Where-Object pullRequestId -eq \
+                                  $env:APPVEYOR_PULL_REQUEST_NUMBER\
+      )[0].buildNumber\
+    ) { ` \
+      throw \"There are newer queued builds \
+              for this pull request, failing early.\" \
+    }"
+  - cmd: ECHO "Filesystem root:"
+  - ps: "ls \"C:/\""
+  - cmd: ECHO "Installed SDKs:"
+  - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
+
+  # Overwrite the visual-studio environment script with the one from
+  # ogrisel/python-appveyor-demo
+  - ps: "\
```

Comment:
I think we should.  It certainly doesn't hurt having it around, and we could quickly switch back to using it if we ever need to.